cypress/support/limitEnvRuns.js

```diff
@@ -0,0 +1,17 @@
+// Allow tests to skipped on live or test and live.
+// Used currently to stop persian tests running on
+// live due to them not yet being routed in the STMs
+
+// For use before routes are open in test env
```

Comment:
Can you please also move the `describeForEuOnly` into this file?

Reply:
I thought about doing that but considered it clearer to have it separately so you can see in the dir that there's an env limit and a separate consideration for EU specifically

Reply:
Okay. Having that visibility is useful, so I'm happy for that to stay in the separate file.